### ISSUE TITLE
Training and Validation Loss is huge

### ISSUE THREAD

**Issue by arohi2bujji** [maxencefaldor/vae#2]
Hi Max,

The training and validation losses for CelebA dataset is very huge for VAE is it expected for this dataset or for only this. How can we reduce the loss?
